<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FJJDialog</class>
 <widget class="QDialog" name="FJJDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/exe.ico</normaloff>:/exe.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font: 12px &quot;微软雅黑&quot;;}
#FJJDialog{background-color: #EEE;}
</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stacked">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid #ADADAD;
	background-color: #E1E2E2;
	color: #444;
}
QPushButton:hover {
	border: 1px solid #0078D7;
	background-color: #E5F1FB;
	color: #024;
}
QPushButton:pressed {
	border: 1px solid #00559B;
	background-color: #CCE4F7;
	color: #024;
}
QPushButton:!enabled {
	border: 1px solid #BFBFBF;
	background-color: #CCCCCC;
	color: #888888;
}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_open">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stacked_open">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_openType">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QPushButton" name="Button_openFile">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="text">
              <string>打开本地文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Button_openGame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="text">
              <string>从游戏目录的数据包中选择dat文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Button_open300RB">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="text">
              <string>从300资源浏览器的导出目录中选择dat文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Button_openCancel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_openList">
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,1,0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_open">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>从游戏目录的数据包中选择dat文件:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combo_open">
             <property name="styleSheet">
              <string notr="true">QComboBox {
	border: 1px solid #ADADAD;
	background-color: #FEFEFE;
	color: #222;
}
QComboBox:drop-down {
	width: 10px;
	border-left: 1px solid #ADADAD;
	background-color: #E1E2E2;
	color: #444;
}
QComboBox:hover {
	border: 1px solid #0078D7;
	background-color: #E5F1FB;
	color: #024;
}
QComboBoxn:pressed {
	border: 1px solid #00559B;
	background-color: #CCE4F7;
	color: #024;
}
QComboBox:!enabled {
	border: 1px solid #BFBFBF;
	background-color: #CCCCCC;
	color: #888888;
}
QScrollBar{width:10px;height:10px;}
QScrollBar:sub-line:vertical {height: 0px;}
QScrollBar:add-line:vertical {height: 0px;}
QScrollBar:sub-line:horizontal{width: 0px;}
QScrollBar:add-line:horizontal{width: 0px;}
QScrollBar:vertical { 
        padding-top:0px;
        padding-bottom:0px;    
        padding-left:0px;    
        padding-right:0px;    
        border-left:0px solid #dddddd;
}
QScrollBar::handle:vertical {
        background:#cccccc;
        min-height:30px;
        border-radius:4px;
}
QScrollBar::handle:vertical:hover {
        background:#999999;
}
QScrollBar:horizontal { 
        padding-top:0px;
        padding-bottom:0px;    
        padding-left:0px;    
        padding-right:0px;    
        border-left:0px solid #dddddd;
}
QScrollBar::handle:horizontal {
        background:#cccccc;
        min-width:30px;
        border-radius:4px;
}
QScrollBar::handle:horizontal:hover {
        background:#999999;
}</string>
             </property>
             <item>
              <property name="text">
               <string>(路径导出)athvalueadjust_newranktime_c.dat</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>新建项目</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="Button_openListCancel">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>取消</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Button_openListOK">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>确定</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_save">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="Button_saveDPK">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>dpk文件（动态补丁使用）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_saveDAT">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>dat文件（保存整个文件）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_saveCancel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_index">
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,1,1,1">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="Button_index0">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>模式 0：第一列整数索引</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_index1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>模式 1：第一、二列索引</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_index2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>模式 2：第一、三列索引</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_index3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>模式 3：第一列字符索引</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_indexCancel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
